evennia/utils/tests/test_funcparser.py

```diff
@@ -437,20 +437,25 @@ def test_callable(self, string, expected):
             ("$You() $conj(smile) at $You(char1).", "You smile at You.", "Char1 smiles at Char1."),
             ("$You() $conj(smile) at $You(char2).", "You smile at Char2.", "Char1 smiles at You."),
             (
-                "$You(char2) $conj(smile) at $you(char1).",
-                "Char2 smile at you.",
-                "You smiles at Char1.",
+                    "$You(char2) $conj(smile) at $you(char1).",
```

Comment:
You really need to fix your auto-formatter, dude. We don't use 8 character indents in Evennia (nor in PEP8). These PRs become unnecessarily spammy this way.